src/sources/kubernetes_logs/mod.rs

```diff
@@ -123,50 +153,69 @@ pub struct Config {
 
     /// Ignore files with a data modification date older than the specified number of seconds.
     #[serde(default)]
+    #[configurable(metadata(docs::type_unit = "seconds"))]
     ignore_older_secs: Option<u64>,
 
     /// Max amount of bytes to read from a single file before switching over
     /// to the next file.
+    ///
     /// This allows distributing the reads more or less evenly across
     /// the files.
+    #[configurable(metadata(docs::type_unit = "bytes"))]
     max_read_bytes: usize,
 
-    /// The maximum number of bytes a line can contain before being discarded. This protects
-    /// against malformed lines or tailing incorrect files.
+    /// The maximum number of bytes a line can contain before being discarded.
+    ///
+    /// This protects against malformed lines or tailing incorrect files.
+    #[configurable(metadata(docs::type_unit = "bytes"))]
     max_line_bytes: usize,
```

Comment:
Looks like these are missing default values (from the existing docs)

Reply:
https://deploy-preview-15724--vector-project.netlify.app/docs/reference/configuration/sources/kubernetes_logs/#max_line_bytes the default is rendered?

Reply:
👀 
TIL ... the default can be rendered from the `Impl` of `Default` ... I've been adding the "serde default = ...."  for each explicitly, even when the Default impl uses the helper function.
I guess it adds clarity to do that but isn't really necessary.

Reply:
I _think_ that's the preferred way, but trying to avoid too many changes ya know